client/package.json

```diff
@@ -2,14 +2,15 @@
   "dependencies": {
     "node-polyglot": "2.0.0",
     "normalize.css": "5.0.0",
-    "vue": "^2.1.0",
-    "vue-resource": "1.0.3",
-    "vue-router": "2.0.1"
+    "preact": "^6.4.0",
+    "preact-compat": "^3.9.2",
```

Comment:
If we want to use React libs like react-router and/or stay compatible with other JSX-compatible lib like inferno, it'd be better to use preact-compat and rewrite our imports for react.